tests/src/ApiTest.php

```diff
@@ -205,15 +205,10 @@ public function testGetSendUri()
      */
     public function testSendWrongJson()
     {
-        $apiMock = $this->getMock(
-            'Openbuildings\Postmark\Api',
-            array(
-                'getSendUri'
-            ),
-            array(
-                'POSTMARK_API_TEST'
-            )
-        );
+        $apiMock = $this->getMockBuilder('Openbuildings\Postmark\Api')
+            ->setMethods(array('getSendUri'))
```

Comment:
@EZarkov This library still supports PHP 5.3. There's no good reason to drop support unless for shorter array syntax. Even if we wanted to drop 5.3 support, that would be a breaking change and would require a version bump.